src/tjenesteomrader/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Layout from 'src/layout';
+import style from './index.module.css';
+import digitalTjenesteBlob from './images/indexBlobs/digitalTjeneste.svg';
+import serviceBlob from './images/indexBlobs/strategi.svg';
+import datadrevet from './images/indexBlobs/datadrevet.svg';
+import kultur from './images/indexBlobs/kultur.svg';
+import Entrance from './components/entrance/entrance';
+
+const Tjenesteomrader = () => {
+  return (
+    <Layout>
+      <div className={style.tjenesteomrader}>
+        <h2>Våre tjenesteområder</h2>
+        <p>
+          Risus, nulla dolor consequat ut ultrices. Egestas eget vitae, tellus
+          volutpat nisi, volutpat diam tincidunt viverra. Sagittis, sapien porta
+          non imperdiet aliquam, eu nec. Enim vel at diam massa tortor blandit
+          et commodo risus. Faucibus lacus metus eget eu eget convallis
+          pharetra.
+        </p>
+        <Entrance
+          blobPath={digitalTjenesteBlob}
+          serviceName={'Digital tjeneste-og produktutvikling'}
+          serviceText={
+            'Risus, nulla dolor consequat ut ultrices. Egestas eget vitae, tellus volutpat nisi, volutpat diam tincidunt viverra.'
+          }
+          buttonColor={'#FFFFFF'}
+          buttonBgColor={'#E61A6B'}
+          urlPath={'digitalTjeneste'}
+        />
+        <Entrance
+          blobPath={serviceBlob}
+          serviceName={'Strategi'}
+          serviceText={
+            'Risus, nulla dolor consequat ut ultrices. Egestas eget vitae, tellus volutpat nisi, volutpat diam tincidunt viverra.'
+          }
+          buttonColor={'#333333'}
+          buttonBgColor={'#03DAC6'}
+          urlPath={'strategi'}
+        />
+        <Entrance
+          blobPath={datadrevet}
+          serviceName={'Datadrevet'}
+          serviceText={
+            'Risus, nulla dolor consequat ut ultrices. Egestas eget vitae, tellus volutpat nisi, volutpat diam tincidunt viverra.'
+          }
+          buttonColor={'#FFFFFF'}
+          buttonBgColor={'#423D89'}
```

Comment:
Sjekk ut theme.module.css for farger her (enten i styleguide eller via `/node_modules/@variant/profile/lib/colors/index.module.css`. Der har du navngitte farger som kan brukes:


Eksempler:
```
--color-primary
--color-secondary1
```

Osv. Fordelen med å bruke de er for det er definert tilsvarende for tekst `--color-secondary1--text` f.eks. Der vil tekstfargen velges ut automatisk for å tilfredsstille UU-krav. Føler også knappene mangler indikasjon på interaksjon her. Type hover og potensielt active (når det trykkes ned). Dette kan du gjøre med å f.eks sende inn farge og suffixe med `__tint1` (eksempel `--color-secondary1__tint1` og `--color-secondary1__tint1--text`).

Du kan faktisk også hente ut disse via TypeScript. Dessverre ikke dokumentert det (enda 😬 ) men du kan gjøre f.eks noe som (pseudokode)

```
import type {ValidColor} from "@variant/profile/colors";
```

Der kan du og hente ut verktøy for a sette rett fargetekst, tint/shades osv. Utforsk import statements der.

Tenker altså API-et her kan gjøres enklere med typesikringen men også at det er fint å ha hover/active states på lenkene.

Reply:
Nice! Visste ikke at vi hadde det